src/provider/tsconfig.json

```diff
@@ -2,7 +2,8 @@
     "extends": "../../tsconfig.json",
     "compilerOptions": {
         "experimentalDecorators": true,
-        "emitDecoratorMetadata": true
+        "emitDecoratorMetadata": true,
+        "preserveConstEnums": true
```

Comment:
This doesn't make any behavioural difference, right? Should this be a services-as-a-whole level decision, rather than a project decision?